test/fixtures/multiple-frames.child-frame.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <title>multiple-frames.child-frame</title>
+  <style>
+  </style>
+</head>
+<body>
+<h1>Texto en un marco infantil.</h1>
```

Comment:
maybe we can use different html for the frame just in case?

Reply:
The text to translate is different in the different frames, albeit not by very much, but enough to distinguish the different frames' html contents from each other. Let me know if I misunderstood your comment, I can follow up in a subsequent PR with improvements :)